Example/Sonar.xcodeproj/project.pbxproj

```diff
@@ -512,6 +514,7 @@
 			baseConfigurationReference = 4F8E94279BF27B78A880A6AE /* Pods-Sonar_Tests.debug.xcconfig */;
 			buildSettings = {
 				BUNDLE_LOADER = "$(TEST_HOST)";
+				DEVELOPMENT_TEAM = S2P73LZ2ZH;
```

Comment:
I guess this small change should not have been commited.

```suggestion
```

Reply:
Oh, sorry!